### PR TITLE
Update search-concept-aggregation.md

### PR DESCRIPTION
**Pull request by ZChristine**
Enrich aggregation filter format for the scenario that multiple values for the same filter are required

### CODE REVIEW

File: api-reference/beta/resources/search-api-overview.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Search requests run on behalf of the user. Search results are scoped to enforce
 |[Use KQL in query terms](#keyword-query-language-kql-support) | **query** |
 |[Sort search results](#sort-search-results)| **sort** |
 |[Refine results using aggregations](#refine-results-using-aggregations)| **aggregations** |
-|[Search across Graph connectors](/graph/search-concept-custom-types)| **contentSources** |
+|[Search Graph connectors](/graph/search-concept-custom-types)| **contentSources** |
```
Comment (Lauragra) [inline api-reference/beta/resources/search-api-overview.md:37]: This title doesn't match the title of the link target. We also appear to have the same link in this topic with slightly different link titles. If we want to keep both links, please change the link text to "Search custom types imported using connectors".

File: api-reference/beta/resources/searchrequest.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ The following is a JSON representation of the resource.
 - Search [mail messages](/graph/search-concept-messages)
 - Search [calendar events](/graph/search-concept-events)
 - Search content in SharePoint and OneDrive ([files, lists and sites](/graph/search-concept-files))
-- Search [custom types (Graph Connectors)](/graph/search-concept-custom-types) data
+- Search [Graph Connectors)](/graph/search-concept-custom-types) data
```
Comment (Lauragra) [inline api-reference/beta/resources/searchrequest.md:75]: Same comment here - the link title should be "Search custom types imported using connectors".

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please update the link text per my comments.

File: api-reference/beta/resources/search-api-overview.md (hunk 1 above)
Comment (nmoreau) [reply api-reference/beta/resources/search-api-overview.md:37]: Fixed

**Review by Lauragra — APPROVED** [review]

**Pull request merged**